src/components/MainContent/MainContent.storybook.tsx

```diff
@@ -0,0 +1,163 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import centered from '@storybook/addon-centered';
+
+import { Colors } from '../../constants';
+import { MainContent, Target } from './index';
+
+const targetMock: Target = {
```

Comment:
I would move this mock into a `__mocks__` folder in this directory, just to remove a big chunk of visual noise.